src/models/service/chains.rs

```diff
@@ -28,3 +30,17 @@ pub struct Theme {
     pub text_color: String,
     pub background_color: String,
 }
+
+#[derive(Serialize, Debug, PartialEq, Clone)]
+#[serde(tag = "type")]
+#[serde(rename_all = "lowercase")]
```

Comment:
Clients generally expect enum variants as `SCREAMING_SNAKE_CASE`

Reply:
Good catch. The config service exposes this as a lower case but this can be different when serialising